tests/Unit/inc/classes/subscriber/CDN/RocketCDN/DataManagerSubscriber/maybeDisableCDN.php

```diff
@@ -23,7 +23,7 @@ public function testShouldReturnNullWhenSubscriptionRunning() {
 		Functions\expect( 'wp_schedule_single_event' )
 			->once();
 
-		$api = $this->createMock( \WP_Rocket\CDN\RocketCDN\APIClient::class );
+		$api = $this->createMock( \WP_Rocket\Engine\CDN\RocketCDN\APIClient::class );
```

Comment:
We've standardized on Mockery. Why? Because Mockery is used as a base dependency for Brain Monkey. In using it, our tests for functions and objects remain consistent.

Let's change all `createMock` to use `Mockery::mock()` instead. This will change the assertions too.